src/main/resources/application.yml

```diff
@@ -23,6 +23,11 @@ zeebe:
 #    server-cert-check: false
 #    enabled: true
 
+spring:
+  main:
+    web-application-type: none
+  profiles:
+    active: "bb,fin12"
```

Comment:
the active profiles is controlled by another configuration , please check the deployment yaml to confirm that this is not creating an additional config

Reply:
removed